rmgpy/data/kinetics/family.py

```diff
@@ -329,10 +329,22 @@ def _apply(self, struct, forward, unique):
                         atom1.apply_action(['CHANGE_BOND', label1, info, label2])
                         atom2.apply_action(['CHANGE_BOND', label1, info, label2])
                         bond.apply_action(['CHANGE_BOND', label1, info, label2])
+                        if pattern:
+                            if bond.is_van_der_waals():
+                                if atom1.is_surface_site():
+                                    atom1.atomtype = [ATOMTYPES['Xv']]
+                                else:
+                                    atom2.atomtype = [ATOMTYPES['Xv']]
```

Comment:
We need to change the `atomtype` from `Xo` to `Xv` if bond is order 0 (van_der_waals)